griptape/tools/tool_output_processor/tool.py

```diff
@@ -1,9 +1,52 @@
 from __future__ import annotations
-from attr import define
-from griptape.mixins import ToolMemoryActivitiesMixin
+from attr import define, field
+from schema import Schema, Literal
+from griptape.artifacts import TextArtifact, ErrorArtifact
 from griptape.tools import BaseTool
+from griptape.utils.decorators import activity
 
 
 @define
-class ToolOutputProcessor(BaseTool, ToolMemoryActivitiesMixin):
-    ...
+class ToolOutputProcessor(BaseTool):
```

Comment:
What do you think about naming it `ToolMemoryClient`?

Reply:
Good idea.